petab/lint.py

```diff
@@ -12,6 +12,10 @@
 
 from . import core
 from . import sbml
```

Comment:
core & sbml seem to be not necessary anymore

Reply:
For me it is. `sbml` is used e.g. in L372

Reply:
372 is empty for me

Reply:
Changed to 374 meanwhile: https://github.com/ICB-DCM/PEtab/blob/fa02643a545ceda765ce696b36b2b594b19dc1e1/petab/lint.py#L374

Reply:
ok, core in L493. sbml I still cannot find